src/shared/dissect-image.c

```diff
@@ -2250,8 +2250,8 @@ int dissected_image_acquire_metadata(DissectedImage *m) {
                 [META_HOSTNAME]          = "/etc/hostname\0",
                 [META_MACHINE_ID]        = "/etc/machine-id\0",
                 [META_MACHINE_INFO]      = "/etc/machine-info\0",
-                [META_OS_RELEASE]        = ("/etc/os-release\0"
-                                           "/usr/lib/os-release\0"),
+                [META_OS_RELEASE]        = "/etc/os-release\0"
+                                           "/usr/lib/os-release\0",
```

Comment:
these are necessary to fix the build with clang 12, see:

https://github.com/systemd/systemd/pull/18470
https://reviews.llvm.org/D85545

Reply:
but now the clang 12 CI is passing, what the... @mrc0mmand any idea what changed?

Reply:
That's... interesting. I have no idea what's changed, but since the clang-12 is still an RC, I guess they fixed it? ~~Let's hope it stays this way~~

Reply:
well now the CI is running, if it starts failing again we'll know what it is and how to fix it

Reply:
either way there should be a comment in code explaining this, because it looks very weird without explanation